package.json

```diff
@@ -44,7 +44,7 @@
     "redux": "3.5.1",
     "redux-immutable": "3.0.6",
     "redux-logger": "2.6.1",
-    "request": "2.71.0",
+    "request": "2.65.0",
```

Comment:
`request` > 2.65 fails with https://github.com/request/request/issues/1920 in webpack because it depends on json-schema which uses a non-standard (incorrect) amd export which apparently didn't get fixed yet.